### DISCUSSION TITLE
installing ImageMagick on Ubuntu

### DISCUSSION THREAD

**Discussion by DennisGCronin** [ImageMagick/ImageMagick#7570] [General] [answered]
I've installed support libs, run "apt install imagemagick", completes successfully. but not finding magick anywhere.

Also, bonus points for an example makefile. I've found a couple examples to use as starting points using ImageMagick (or MagickWand I guess) from C but can't figure out how to compile them.  Thanks.

Dennis

**Comment by rich2005** [comment, marked as answer]
Just for info: Ubuntu (24.04) uses IM 6.9 and the convert command.

For magick there is the (portable) appimage version 7.something and what I do is make a symlink to that.
![magick](https://github.com/user-attachments/assets/a1fae312-dd83-477e-b86c-1636bc3bbb04)

Edit: I used to compile a static version which acts just the same as the appimage, needs all the imageformat .dev files of course. Pulled this out, last used with ubuntu 20

```
wget http://www.imagemagick.org/download/ImageMagick.tar.gz

tar xzvf ImageMagick.tar.gz && cd cd ImageMagick-7.0.*/

./configure --prefix='/home/rich/magick' --disable-shared --enable-delegate-build --with-webp=yes --with-heic=no


make
make install
```
